services/spire/federation/clusterfederatedtrustdomain-cluster1.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: spire.spiffe.io/v1alpha1
+kind: ClusterFederatedTrustDomain
+metadata:
+  name: cluster2.demo
+spec:
+  trustDomain: cluster2.demo
+  bundleEndpointURL: https://172.16.17.3:8443 #replace with ip of spire server endpoint on cluster 2
+  bundleEndpointProfile:
+    type: https_spiffe
+    endpointSPIFFEID: spiffe://cluster2.demo/spire/server
+  trustDomainBundle: |- #replace keys with trust bundle from cluster 2
+    {
+      "keys": [
+        {
+            "use": "x509-svid",
+            "kty": "EC",
+            "crv": "P-256",
+            "x": "7ZZmZoyTep5ssVTmZMw3sMQUjkOdxQTq2b0R5rTVlHM",
+            "y": "7BiKMky421_ggato-vff6Wv5UaJZm4CaGsp-t2DvFx4",
+            "x5c": [
```

Comment:
Mask the keys. Replace it with the palceholder like <Get your keys from cluster using command...>